client/components/bulk-select/index.jsx

```diff
@@ -43,20 +43,32 @@ export default class extends React.Component {
 	};
 
 	render() {
+		const { translate, ariaLabel = translate( 'Select All' ) } = this.props;
 		const isChecked = this.hasAllElementsSelected();
 		const inputClasses = classNames( 'bulk-select__box', {
 			// We need to add this CSS class to be able to test if the input if checked,
 			// since Enzyme still doesn't support :checked pseudoselector.
 			'is-checked': isChecked,
 		} );
+
 		return (
-			<span className="bulk-select" onClick={ this.handleToggleAll }>
-				<span className="bulk-select__container">
-					<input type="checkbox" className={ inputClasses } checked={ isChecked } readOnly />
+			<span className="bulk-select">
+				{ /* eslint-disable jsx-a11y/label-has-for */ }
```

Comment:
This is because you're using `aria-label` below, right?

Reply:
No, I guess I should add a proper comment. You can omit the `htmlFor` on nested inputs, because there's an implicit relationship between the label & input, but we seem to have that part of the rule turned off (and so always require the explicit relationship from htmlFor).